app/views/users/summary.slim

```diff
@@ -58,15 +49,15 @@ div.content-container
         div.form-group.has-feedback
           div.input-group
             label.input-group-addon.input-min-width for="summaryTimeStart" Start Date
-            input.form-control.tooltips id="summaryTimeStart" type="date" name="min_date" placeholder="1999-12-31" title="Start Date" required="required" value="#{params[:min_date]}"
+            input.form-control.tooltips id="summaryTimeStart" type="date" name="min_date" title="Start Date" value="#{params[:min_date]}"
           span.glyphicon.form-control-feedback aria-hidden="true"
         div.form-group.has-feedback
           div.input-group
             label.input-group-addon.input-min-width for="summaryTimeEnd" End Date
-            input.form-control.tooltips id="summaryTimeEnd" type="date" name="max_date" placeholder="2099-12-31" title="End Date" required="required" value="#{params[:max_date]}"
+            input.form-control.tooltips id="summaryTimeEnd" type="date" name="max_date" title="End Date" value="#{params[:max_date]}"
           span.glyphicon.form-control-feedback aria-hidden="true"
         div.button-container
-          a tabindex="-1"
+          a href="#{summary_user_path(@viewed_user)}" tabindex="-1"
+            button.btn.btn-primary type="reset" Reset
+          a id="filterUserSummarySubmit" tabindex="-1"
```

Comment:
So resetting does work but not if you have submitted before.

Submitting changes the URL and sets the values equal to what was preloaded into the URL. Resetting only clears new values, not the values that were present at the initial state of the form (which was after last submit). You can clear before you hit submit but not after if you expect the whole thing to clear.

Think we need to ditch the native browser functionality and bring back the JS we had.